src/libaktualizr/uptane/uptane_network_test.cc

```diff
@@ -203,7 +203,7 @@ TEST(UptaneNetwork, LogConnectivityRestored) {
   http->connectSwitch = true;
   testing::internal::CaptureStdout();
   result = up->fetchMeta();
-  EXPECT_EQ(result.status, result::UpdateStatus::kUpdatesAvailable);
+  EXPECT_EQ(result.status, result::UpdateStatus::kError);
```

Comment:
This doesn't seem right. Was something going wrong here?

Reply:
Nothing's wrong. Just make a failure case to test the GitLab report feature.